cloud_registry/__init__.py

```diff
@@ -1 +0,0 @@
-__version__ = '0.1.0'
```

Comment:
Sorry to nag, but can you include a module-level docstring, like the one here?
https://github.com/elixir-cloud-aai/proTES/blob/dev/pro_tes/__init__.py

So basically,

```py
"""Cloud Registry app package root."""
```